### ISSUE TITLE
Chart update recommendations

### ISSUE THREAD

**Issue by aarnq** [elastisys/compliantkubernetes-apps#673]
The results from #649 which does not need immediate attention, as well as some cleanup.

- [x] charts / examples / user-alertmanager
  - Move out of examples.
- [x] charts / falcosidekick
  - Remove after it has been disabled, it's going to be a subchart after #650.
- [x] charts / gatekeeper-operator
  - Remove, it's been disabled since v0.12.0 after the official chart was added.
- [ ] charts / sc-logs-retention
  - Potentially update the amazon/aws-cli image.
- [ ] charts / vulnerability-exporter
  - Potentially update the prom/node-exporter image.
- [x] upstream / fluentd
  - v1.11.4/v2.4.0 (2020/11/06) -> v1.14.2/v4.3.4 (2021/10/29)
  - Updates fluentd and dependencies which seem to solve a few performance issues regarding high CPU usage and buffer overflows.
- [x] upstream / fluentd-elasticsearch
  - v3.1.0/v10.2.1 (2020/10/30) -> v3.3.3/v13.0.0 (2021/10/27)
  - Updates fluentd to v1.12.0 with updated dependencies. New features, and new configuration options in place of deprecated ones.
- [ ] upstream / harbor
  - v2.2.1/v1.6.1 (2021/03/31) -> v2.4/v1.8.0 (2021/11/03)
  - Lots of general improvements and bug fixes, for example improvements for how OIDC based users are handled and the support to delete them.
  - Note that the current version is impacted by [a bug](https://github.com/goharbor/harbor/issues/14932) in notary where keys created in this version will be incompatible when upgrading, there is [a workaround](https://github.com/goharbor/harbor/wiki/Harbor-FAQs#notary-key-not-found) for it.

**Comment by Xartos** [comment]
Closing in favor of the issue that will be created from https://github.com/elastisys/compliantkubernetes-apps/issues/998
